action_server/src/action_server/actions/follow.py

```diff
@@ -76,6 +76,9 @@ def _configure(self, robot, config):
 
         self._robot = robot
 
+        self._follow_sm = FollowOperator(self._robot)
+
+        self.cancel_requested = False
```

Comment:
not used anywhere. correct?

Reply:
You're totally right

Reply:
What is the function of it? Any future plans?

Reply:
No, I'll remove it. But didn't get the chance yet.